index.js

```diff
@@ -5,6 +5,10 @@ const console = require('console')
 const extractPluginName = require('./stackParser')
 
 function plugin (fn, options = {}) {
+  if (typeof fn.default !== 'undefined') {
```

Comment:
Can you add a comment to explain the reason for this check? :)

Reply:
If I understood correctly, it is for supporting `export default ....` behaviour

Reply:
@cemremengu exactly, this is for posterity :)